pages/moderation/index.vue

```diff
@@ -70,8 +92,14 @@
         slot-scope="scope"
       >
         <nuxt-link :to="{ name: 'profile-slug', params: { slug: scope.row.reporter.slug } }">
-          {{ scope.row.reporter.name }}
+          {{ scope.row.reporter.name || 'Anonymus' | truncate(26) }}
```

Comment:
`to.many.dots.make.a.NullPointerException.very.likely`

Reply:
what do you mean 😄